test/sequential/test-debugger-sb-before-load.js

```diff
@@ -10,35 +10,31 @@ const assert = require('assert');
 const path = require('path');
 
 // Using sb before loading file.
-{
-  const scriptFullPath = fixtures.path('debugger', 'cjs', 'index.js');
-  const script = path.relative(process.cwd(), scriptFullPath);
-
-  const otherScriptFullPath = fixtures.path('debugger', 'cjs', 'other.js');
-  const otherScript = path.relative(process.cwd(), otherScriptFullPath);
-
-  const cli = startCLI([script]);
-
-  function onFatal(error) {
-    cli.quit();
-    throw error;
-  }
-
-  cli.waitForInitialBreak()
-    .then(() => cli.waitForPrompt())
-    .then(() => cli.command('sb("other.js", 2)'))
-    .then(() => {
-      assert.match(
-        cli.output,
-        /not loaded yet/,
-        'warns that the script was not loaded yet');
-    })
-    .then(() => cli.stepCommand('cont'))
-    .then(() => {
-      assert.ok(
-        cli.output.includes(`break in ${otherScript}:2`),
-        'found breakpoint in file that was not loaded yet');
-    })
-    .then(() => cli.quit())
-    .then(null, onFatal);
-}
+
+const scriptFullPath = fixtures.path('debugger', 'cjs', 'index.js');
+const script = path.relative(process.cwd(), scriptFullPath);
+
+const otherScriptFullPath = fixtures.path('debugger', 'cjs', 'other.js');
+const otherScript = path.relative(process.cwd(), otherScriptFullPath);
+
+const cli = startCLI([script]);
+
+(async () => {
+  await cli.waitForInitialBreak();
+  await cli.waitForPrompt();
+  await cli.command('profile');
+  await cli.command('sb("other.js", 2)');
+  assert.match(cli.output, /not loaded yet/,
+               'warns that the script was not loaded yet');
+  await cli.command('profileEnd');
+  assert.match(cli.output, /\[Profile \d+μs\]/);
+  await cli.command('profiles');
+  assert.match(cli.output, /\[ \[Profile \d+μs\] \]/);
+  await cli.command('profiles[0].save()');
+  assert.match(cli.output, /Saved profile to .*node\.cpuprofile/);
+  await cli.stepCommand('cont');
+  assert.ok(cli.output.includes, `break in ${otherScript}:2`,
+            'found breakpoint in file that was not loaded yet');
```

Comment:
This now checks that `cli.output.includes` is `true`-ish, but it is a function so it will always be `true`-ish. Instead, call the function and check that the return value is `true`-ish as in the original:

```suggestion
  assert.ok(cli.output.includes(`break in ${otherScript}:2`),
            'found breakpoint in file that was not loaded yet');
```